ExilenceNextApp/src/components/toolbar/ToolbarContainer.tsx

```diff
@@ -78,6 +83,27 @@ const ToolbarContainer: React.FC<ToolbarContainerProps> = ({
     });
   };
 
+  const handleLogMonitor = () => {
+    logStore!.running
```

Comment:
We have `logStore?: LogStore` which indicates that this will be optional or not initialized at the start. 
Not sure if providing `!.` operator is intended here and if we should not just do a simple check for logStore and stick to `key.value` instead of `key!.value`

```const handleLogMonitor = () => {
    if(!logStore) returm;
    logStore.running
   ...
```
WDYT @viktorgullmark?

Reply:
This is the current way we instantiate stores, but all of this will be changed when I add the `useStores` hook I plan on building soon. Not a big issue for now as this will entirely change when its added in imo